doma-spring-boot-samples/doma-spring-boot-sample-testcontainers/src/test/java/org/seasar/doma/boot/sample/ApplicationTest.java

```diff
@@ -4,49 +4,56 @@
 
 import java.util.List;
 
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.test.context.SpringBootTest;
 import org.springframework.boot.test.context.SpringBootTest.WebEnvironment;
-import org.springframework.boot.test.web.client.TestRestTemplate;
 import org.springframework.boot.test.web.server.LocalServerPort;
 import org.springframework.context.annotation.Import;
 import org.springframework.core.ParameterizedTypeReference;
-import org.springframework.http.HttpEntity;
-import org.springframework.http.HttpMethod;
+import org.springframework.web.client.RestClient;
 import org.springframework.web.util.UriComponentsBuilder;
 
 @SpringBootTest(webEnvironment = WebEnvironment.RANDOM_PORT)
 @Import(TestcontainersConfiguration.class)
 class ApplicationTest {
-	@Autowired
-	private TestRestTemplate restTemplate;
+	private RestClient restClient;
 	private final ParameterizedTypeReference<List<Message>> typedReference = new ParameterizedTypeReference<List<Message>>() {
 	};
 	@LocalServerPort
 	private int port;
 
+	@BeforeEach
+	void setUp(@Autowired RestClient.Builder restClientBuilder) {
+		this.restClient = restClientBuilder.defaultStatusHandler(__ -> true, (req, res) -> {
+		}).build();
```

Comment:
use `.baseUrl("http://localhost:" + port)` to build the `RestClient` and use `.uri("/?text={text}", "hello")` for each test method